src/Message/CaptureRequest.php

```diff
@@ -62,103 +71,64 @@ protected function getResponseName(): string
     /**
      * Check all payments to see if we have an authorized one
      *
-     * @param array $payments
+     * @param stdClass[] $payments
      *
-     * @return array
+     * @return stdClass|null
      */
-    protected function getAllAuthorizedPayments(array $payments): array
+    private function getPaymentAvailableForCapture(array $payments): ?stdClass
     {
-        $authorizedPayments = [];
+        $payment = null;
```

Comment:
I actually did this on purpose. This way when there is at least one payment it is always returned, even though it might already be captured. CM Payment will respond with the right error message.

Though, I missed elaborating on this in the docblock. I will update the docblock.